lib/Elastica/Bulk/ResponseSet.php

```diff
@@ -42,16 +42,13 @@ public function getBulkResponses()
      */
     public function getError()
     {
-        $error = '';
-
         foreach ($this->getBulkResponses() as $bulkResponse) {
             if ($bulkResponse->hasError()) {
-                $error = $bulkResponse->getError();
-                break;
+                return $bulkResponse->getError();
```

Comment:
And in this project already see more than one return, in my changes too - change return structure